### PR TITLE
Add WebM to docs

### PR DESCRIPTION
**Pull request by f3ndot**
To be tied to the merge and release of https://github.com/HandBrake/HandBrake/pull/1822

### CODE REVIEW

**Comment by f3ndot** [comment]
5580f36 isn't entirely related to WebM, but I noticed the CLI reference was out of date in a few places. The entire thing was clobbered with the direct output from me running `--help` on latest build

**Comment by f3ndot** [comment]
@bradleysepos might be smart to have a CI master-branch build job that will run `handbrake --help`, capture the output, sed the latest HandBrake-docs, and put up a PR for y'all. Easy enough to do with a python script. Might spare you some of the menial tasks with syncing these repos.

Anyway. I'm now happy with this PR to merge as-is.

**Comment by f3ndot** [comment]
@bradleysepos polite ping, this is now ready to merge

File: source/docs/en/latest/introduction/about.markdown [hunk 1]
```diff
@@ -20,18 +20,18 @@ About HandBrake
 
 HandBrake is an open-source video transcoder available for Linux, Mac, and Windows. Everyone can use HandBrake to make videos *for free*.
 
-HandBrake is a post-production tool. Its primary purpose is to convert videos from supported [source formats](../technical/source-formats.html) to MP4 or MKV format. If you wish to make changes to your `Source` video, please use an appropriate [video editing software](https://en.wikipedia.org/wiki/List_of_video_editing_software).
+HandBrake is a post-production tool. Its primary purpose is to convert videos from supported [source formats](../technical/source-formats.html) to MP4, MKV or WebM format. If you wish to make changes to your `Source` video, please use an appropriate [video editing software](https://en.wikipedia.org/wiki/List_of_video_editing_software).
```
Comment (bradleysepos) [inline source/docs/en/latest/introduction/about.markdown]: Oxford Comma, you monster! 😆

**Comment by bradleysepos** [comment]
Merged with minor revisions. Thanks!

**Pull request merged**